NYPLAudiobookToolkit/UI/AudiobookPlayerViewController.swift

```diff
@@ -483,6 +496,21 @@ let SkipTimeInterval: Double = 15
             }
         }
     }
+    
+    private func updateColors() {
+        if #available(iOS 12.0, *),
+           UIScreen.main.traitCollection.userInterfaceStyle == .dark {
+            // Use solid color background for Dark Mode
+            self.gradient.colors = [ NYPLColor.primaryBackgroundColor.cgColor ]
+        } else {
+            let startColor = UIColor(red: (210 / 255), green: (217 / 255), blue: (221 / 255), alpha: 1).cgColor
+            self.gradient.colors = [ startColor, UIColor.white.cgColor ]
+        }
+        self.gradient.layoutIfNeeded()
```

Comment:
would it work if instead of forcing layout we did a `setNeedsLayout`, or even just `setNeedsDisplay`?

Reply:
Nice catch, fixed